resources/rebuild.sh

```diff
@@ -0,0 +1,179 @@
+#!/bin/bash
+# Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# fail if we encounter an error, uninitialized variable or a pipe breaks
+set -eu -o pipefail
+
+set -x
+PS4='+\t '
+
+cd $(dirname $0)
+ARCH=$(uname -m)
+OUTPUT_DIR=$PWD/$ARCH
+
+# Make sure we have all the needed tools
+function install_dependencies {
+    sudo apt update
+    sudo apt install -y bc flex bison gcc make libelf-dev libssl-dev squashfs-tools busybox-static tree cpio curl
```

Comment:
nit: Can we remove dependencies for kernel build from our devctr?
https://github.com/firecracker-microvm/firecracker/blob/fd72d1aaa74633c6f0adadd8114853990382d86e/tools/devctr/Dockerfile#L54-L57C13

Reply:
Good idea